Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -3805,6 +3805,11 @@ - (void)_setFontForWebView:(UIFont *)font sender:(id)sender
     changes.setFontSize(font.pointSize);
     changes.setBold(font.traits & UIFontTraitBold);
     changes.setItalic(font.traits & UIFontTraitItalic);
+
+    NSString *styleAttribute = [font.fontDescriptor.fontAttributes objectForKey:UIFontDescriptorTextStyleAttribute];
```

Comment:
NIT: maybe we would have this be named something more specific like `fontDescriptor` instead of this very general name?

also, i think you can inline this
```
if (NSString *fontDescriptor = [font.fontDescriptor.fontAttributes objectForKey:UIFontDescriptorTextStyleAttribute])
    changes.setFontFamily(fontDescriptor);
```

Reply:
It's not the font descriptor though, it's a value of one of the font's descriptor's attributes

Reply:
It's also pretty confusing since there's two meanings of "text style" basically. There's text style as in italics and bolding and then there's this meaning of the system text style attribute thing

Reply:
I could name it `systemTextStyleAttribute`?

Reply:
Yeah, I would call the local variable something like `textStyleAttribute`